go.mod

```diff
@@ -45,9 +45,29 @@ replace (
 	github.com/openshift/hive/pkg/apis => github.com/carbonin/hive/pkg/apis v0.0.0-20210209195732-57e8c3ae12d1
 	github.com/openshift/machine-api-operator => github.com/openshift/machine-api-operator v0.2.1-0.20201026110925-50ea569da51b
 	k8s.io/api => k8s.io/api v0.19.2
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.21.0
 	k8s.io/apimachinery => k8s.io/apimachinery v0.19.2
+	k8s.io/apiserver => k8s.io/apiserver v0.21.0
+	k8s.io/cli-runtime => k8s.io/cli-runtime v0.21.0
 	k8s.io/client-go => k8s.io/client-go v0.19.2
+	k8s.io/cloud-provider => k8s.io/cloud-provider v0.21.0
+	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.21.0
 	k8s.io/code-generator => k8s.io/code-generator v0.0.0-20200214080538-dc8f3adce97c
+	k8s.io/component-base => k8s.io/component-base v0.21.0
+	k8s.io/component-helpers => k8s.io/component-helpers v0.21.0
+	k8s.io/controller-manager => k8s.io/controller-manager v0.21.0
+	k8s.io/cri-api => k8s.io/cri-api v0.21.0
+	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.21.0
+	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.21.0
+	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.21.0
+	k8s.io/kube-proxy => k8s.io/kube-proxy v0.21.0
+	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.21.0
+	k8s.io/kubectl => k8s.io/kubectl v0.21.0
+	k8s.io/kubelet => k8s.io/kubelet v0.21.0
+	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.21.0
+	k8s.io/metrics => k8s.io/metrics v0.21.0
+	k8s.io/mount-utils => k8s.io/mount-utils v0.21.0
+	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.21.0
```

Comment:
That's a lot of replaces... are those mandatory?
cc @eranco74

Reply:
Yes, they are required by OCS

Reply:
We would like to avoid that if possible...
The last time we did it for hive was troublesome
cc @filanov

Reply:
You think that we can suggest OCS to work with sub moduls?

Reply:
Because of https://github.com/kubernetes/kubernetes/issues/79384, we have to update the version

Reply:
Because you are only using utils and api there i don't think that you need all the replaces.
We had similar issue with Hive when we had to replace everything that they replaced and then every component that import from assisted-service did the same, it was very hard to manage. 
So what we did was to add sub modules in Hive, we used only the API so why do we need to import anything else? 
Maybe we need to suggest the same to ocs? 
Here is an example https://github.com/openshift/hive/tree/master/apis